packages/@ama-sdk/schematics/schematics/ng-update/index.ts

```diff
@@ -13,10 +11,12 @@ import {
   updateV10_3 as tsUpdateV10_3,
   updateV11_0 as tsUpdateV11_0
 } from './typescript';
+/* eslint-enable camelcase */
```

Comment:
maybe we can keep it disabled for the whole file ?

Reply:
Yes, as the team prefer